torch/csrc/distributed/rpc/init.cpp

```diff
@@ -270,8 +273,13 @@ If the future completes with an error, an exception is thrown.
           Arguments:
               num_send_recv_threads (int, optional): The number of threads in
                   the thread-pool used by ``ProcessGroupAgent`` (default: 4).
-              rpc_timeout (datetime.timedelta, optional): The timeout for RPC
-                  requests (default: ``timedelta(seconds=60)``).
+              rpc_timeout (datetime.timedelta, optional): The default timeout
```

Comment:
Since we are now doing timeouts with `float` for torchscript reasons, should we change these to also be `float` instead of `timedelta`? If so, I can send a follow up PR doing this. We would have to change the `RpcBackendOptions` and `_set_rpc_timeout` APIs. cc @pritamdamania87 @mrshenli

Reply:
I'd vote for yes, let's make them consistent.

Reply:
Will address in the next diff.